docs/book/src/user/quick-start.md

```diff
@@ -7,10 +7,51 @@ In this tutorial we'll cover the basics of how to use Cluster API to create one
 - Install and setup [kubectl] in your local environment.
 - A Kubernetes cluster to be used as [management cluster].
   For the purpose of this quick start we suggest you to use [kind] as a _non production-ready_ solution.
+
+{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere"}}
+{{#tab AWS}}
+
+  ```bash
+  kind create cluster --name=clusterapi
+  export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+  ```
+{{#/tab }}
+{{#tab Docker}}
+
+<aside class="note warning">
+
+<h1>Not Intended For Production</h1>
+
+Because it requires no external infrastructure, the Docker provider allows you to quickly try Cluster API. It is also useful for developing Cluster API, and tooling that uses it. **Running a production cluster using the Docker provider is not supported**.
+
+</aside>
+
+  Because the Docker provider needs to access Docker on the host, a custom kind cluster configuration is required:
```

Comment:
Do we need a note somewhere that says it doesn't work with Docker for Mac? @chuckha

Reply:
@erwinvaneyk Your thoughts?

Reply:
Both v1alpha1 and v1alpha2 work on docker-for-mac. In both cases you need to adapt the kubeconfig a bit (with v1alpha2 it is a bit more tricky than with v1alpha1). I don't recall the exact steps though, will need to look them up later.

Reply:
Yeah, we need to fix this issue. The steps for os x are to look at the exposed port that maps to the api server on the load balancer and update your kubeconfig. I can provide some instructions today to make this complete

Reply:
note: this is fine to fix after merge

Reply:
I'll leave a comment that MacOS instructions are forthcoming

Reply:
Filed https://github.com/kubernetes-sigs/cluster-api/issues/1534